src/Sulu/Bundle/WebsiteBundle/Resources/views/Sitemap/sitemap.xml.twig

```diff
@@ -7,35 +7,38 @@
         {% if domain in location %}
             <url>
                 {% block url %}
-                    <loc>{{ location }}</loc>
-                    {% if entry.lastmod %}
-                        <lastmod>{{ entry.lastmod|date('Y-m-d') }}</lastmod>
-                    {% endif %}
-                    {% if entry.changefreq %}
-                        <changefreq>{{ entry.changefreq }}</changefreq>
-                    {% endif %}
-                    {% if entry.priority %}
-                        <priority>{{ entry.priority }}</priority>
-                    {% endif %}
+                    {% block loc %}
```

Comment:
I just added more blocks to override the template here and removed the `x-default` behaviour as it is incorrect.